roles/rsyslog/roles/README.md

```diff
@@ -16,8 +16,6 @@ The `defaults` directory includes:
   - It is required to set for all logs the project identfier for pipelining:
     set $.logs_collection = "project name";
     For example: In `ovirt` input role, in the default/main.yaml, for every log `$.logs_collection` is set to `ovirt`.
-  - If rsyslog_default equals to`"true, It is required to set for all logs you don't want to be processed by the default rules:
-    set $.send_targets_only = "true";
```

Comment:
@nhosoi Why do you remove rsyslog_default?

Reply:
@sradco:
> @nhosoi Why do you remove rsyslog_default?

Please see this JIRA card.
https://projects.engineering.redhat.com/browse/RHELPLAN-31390?focusedCommentId=2055981&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2055981

It may be a bit confusing to follow, but first I renamed rsyslog_default with rsyslog_use_roles (having the opposite boolean), then I got a question from @pcahyna why this is needed.  In the discussion, it's concluded since by rerunning yum reinstall, we could achieve the goal, LSR/Logging does not have to do it.  Also, any other LSR subroles do not do that.  It seems it somehow "violating" the LSR policy.

Reply:
@sradco:
> We needed a way to keep the default filters from rsyslog.conf for the system and env logs,

You mean you use the original rsyslog.conf from the rsyslog rpm with the config files in /etc/rsyslog.d/ deployed by the LSR/Logging?  That is, ovirt is deploying config files with the original rsyslog.conf.

> but we needed to stop the other logs, that we sent to elasticsearch for example, from being logged also to journald.

I think the original rsyslog.conf could be replaced with files_forwards output.  Currently, the type is `output`, but it actually contains rules, so it's more appropriate to have `rules` type, which is 50.
https://github.com/linux-system-roles/logging/pull/80/files#diff-c30f197e75c01b82b3a0dda13e1ef238R36